canonical-web-and-design/maas-ui
# Add back ./run

## Done
Add back ./run

## QA
- Run `./run clean`
- Run `./run`
- Run `./run test`
- All the above should be successful
